prefs.js

```diff
@@ -87,17 +119,22 @@ const TransparentWindowPrefsWidget = new GObject.Class({
     LogLevelWidget.connect('changed', function(comboWidget) {
       setting.set_int('verbose-level', comboWidget.get_active());
     });
-    LogLevelBox.pack_start(LogLevelLabel, true, true, 0);
-    LogLevelBox.add(LogLevelWidget);
-    this.add(LogLevelBox);
+
+    if (gnome_at_least_40_1) {
+      LogLevelBox.prepend(LogLevelLabel, true, true, 0);
+      LogLevelBox.append(LogLevelWidget);
+      this.append(LogLevelBox);
+    } else {
+      LogLevelBox.pack_start(LogLevelLabel, true, true, 0);
+      LogLevelBox.add(LogLevelWidget);
+      this.add(LogLevelBox);
+    }
   },
 });
 
 
 function buildPrefsWidget(){
   let widget = new TransparentWindowPrefsWidget();
 
-  widget.show_all();
```

Comment:
please add if statement for this line as well

Reply:
Ok, I've done )